fix(banner): update docs, screenshots to reference new variation names (#3178)
patternfly/patternfly-org: packages/v4/patternfly-docs/content/design-guidelines/components/banner/banner.md

PatternFly offers 5 different banner types detailed below.
 
<img src="./img/banner-types.png" alt="5 different types of banners"/>
 
|Type |Background color |Text color | Usage |
|-|-|-|-|
| Default | `--pf-global--BackgroundColor--dark-400` | White | Generic messages with no associated severity |
| Blue | `--pf-global--info-color--100` | Black | General information messages |
| Red | `--pf-global--danger-color--100` | White | High severity messages
| Green | `--pf-global--success-color--100` | White | Positive, success messages |

> | Gold | `--pf-global--warning-color--100` | Black | Warning or medium severity messages |

Reviewer: Maybe not within the table itself, but perhaps in some context surrounding it, would it be worth mentioning something along the lines of "When used to convey status..." or maybe updating the header to "Usage (status)"/"Usage as status"?

Basically to convey that there are 5 color types and each color has a respective status, but color does not inherently === status

> PatternFly suggests that users adopt one of these 5 colors, as they’ve been tested with their text colors for accessibility. However, if colors outside of these suggestions are used, we encourage using discretion when selecting the text color.

Reviewer: Nit:

```suggestion
PatternFly suggests that users adopt one of these 5 colors, as they’ve been tested with their text colors for accessibility. However, if colors outside of these suggestions are used, we encourage manually checking color contrasts when selecting the text color.
```

May depend on if we want to explicitly mention "color contrast", otherwise "using discretion" is good here

### When to use
 
Banners are flexible and can be used for any content and for several use cases outlined in the following examples.
 
- **To indicate that an application/site is offline.** In this example, a banner is used to indicate to the user that the site is undergoing maintenance and that they will have limited functionality as a result. The banner is shown at the top and bottom to ensure that the user does not miss this disclaimer. Since this is a general, non-severe message, a default banner was used.
 
   <img src="./img/example-1.png" alt="Example of default banner in use" />
 

> - **To indicate that an application upgrade is available.** In this example, the user is notified of an available upgrade through a banner at the top of the page. Since the use case here is to convey information, an blue status banner was used.

Reviewer: ```suggestion
- **To indicate that an application upgrade is available.** In this example, the user is notified of an available upgrade through a banner at the top of the page. Since the use case here is to convey information, a blue status banner was used.
```

### Placement
 
All banners should be a solid bar that spans the top and/or bottom of either:
- The full page
 
   <img src="./img/placement-1.png" alt="Example of banner placement above and below the page" />
 
- The main content area of the page
 
   <img src="./img/placement-2.png" alt="Example of banner placement below above and below the main content area of the page" />
 
In all cases, the banner should be sticky to where it is used.

## Spacing

### Padding
 
The padding required above and below a banner will depend on where it is placed.
 

> - **When placed above or below the page,** the padding above and below should be 0px, as the banner should touch top and/or bottom of the page.

Reviewer: ```suggestion
- **When placed above or below the page,** the padding above and below should be 0px, as the banner should touch the top and/or bottom of the page.
```

 
  <img src="./img/placement-1.png" alt="Padding when banner is placed above and below the page" />
 

> - **When placed above the main content area,** there should be no padding above the banner, and in most cases there should be 24px padding below it. The only exception is when breadcrumbs are directly below the banner &mdash; in that case, the padding should be 16px.

Reviewer: ```suggestion
- **When placed above the main content area,** there should be no padding above the banner, and in most cases there should be 24px padding below it. The only exception is when breadcrumbs are directly below the banner &mdash; in that case, the padding should be 16px below the banner.
```
